app/cdap/components/EntityCard/FastActions/index.js

```diff
@@ -20,6 +20,9 @@ import React, { Component } from 'react';
 import FastAction from 'components/FastAction';
 import { objectQuery } from 'services/helpers';
 import isNil from 'lodash/isNil';
+import { getDataTestid } from '../../../testids/TestidsProvider';
+
+const TESTID_PREFIX = 'features.entityListView.entity.fastActions';
```

Comment:
I see that the datatestid.yaml changes are missing in this PR. Can you please check?

Reply:
Changes are there in the yaml files, probably got merged with other PR. Ignore my previous comment